components/list/list-item-mixin.js

```diff
@@ -0,0 +1,351 @@
+			}
+			d2l-list-item-generic-layout {
+				border-bottom: 1px solid transparent;
+				border-bottom: 1px solid var(--d2l-color-mica);
+				border-top: 1px solid var(--d2l-color-mica);
+			}
+			[slot="content"].d2l-list-item-content-extend-separators {
+				padding-left: 0.9rem;
+				padding-right: 0.9rem;
+			}
+			a[href].d2l-list-item-link {
+				width: 100%;
+				height: 100%;
+			}
+			.d2l-list-item-content ::slotted(*) {
+				margin-top: 0.05rem;
+			}
+			.d2l-list-item-content.d2l-hovering,
+			.d2l-list-item-content.d2l-focusing {
+				--d2l-list-item-content-text-decoration: underline;
+			}
+			[slot="content-action"]:focus {
+				outline: none;
+			}
+			[slot="content"] {
+				justify-content: stretch;
+				padding: 0.55rem 0px;
+				display: flex;
+			}
+			[slot="content"] ::slotted([slot="illustration"]) {
+				margin: 0.15rem 0.9rem 0.15rem 0;
+				max-height: 2.6rem;
+				max-width: 4.5rem;
+				border-radius: 6px;
+				overflow: hidden;
+				flex-grow: 0;
+				flex-shrink: 0;
+			}
+			:host([dir="rtl"]) [slot="content"] ::slotted([slot="illustration"]) {
+				margin-left: 0.9rem;
+				margin-right: 0;
+			}
+			.d2l-list-item-actions-container {
+				padding: 0.55rem 0;
+			}
+			::slotted([slot="actions"]) {
+				display: grid;
+				grid-auto-columns: 1fr;
+				grid-auto-flow: column;
+				gap: 0.3rem;
+				margin: 0.15rem 0;
+			}
+
+			[data-breakpoint="1"] ::slotted([slot="illustration"]),
+			[data-breakpoint="1"] .d2l-list-item-content-flex ::slotted([slot="illustration"]) {
+				margin-right: 1rem;
+				max-height: 3.55rem;
+				max-width: 6rem;
+			}
+			:host([dir="rtl"]) [data-breakpoint="1"] ::slotted([slot="illustration"]) {
+				margin-left: 1rem;
+				margin-right: 0;
+			}
+			[data-breakpoint="2"] ::slotted([slot="illustration"]) {
+				margin-right: 1rem;
+				max-height: 5.1rem;
+				max-width: 9rem;
+			}
+			:host([dir="rtl"]) [data-breakpoint="2"] ::slotted([slot="illustration"]) {
+				margin-left: 1rem;
+				margin-right: 0;
+			}
+			[data-breakpoint="3"] ::slotted([slot="illustration"]) {
+				margin-right: 1rem;
+				max-height: 6rem;
+				max-width: 10.8rem;
+			}
+			:host([dir="rtl"]) [data-breakpoint="3"] ::slotted([slot="illustration"]) {
+				margin-left: 1rem;
+				margin-right: 0;
+			}
+			input[type="checkbox"].d2l-input-checkbox {
+				margin: 1.15rem 0.9rem 1.15rem 0;
+			}
+			d2l-list-item-drag-handle {
+				margin: 0.8rem 0 0.8rem 0.4rem;
+			}
+			:host([dir="rtl"]) input[type="checkbox"].d2l-input-checkbox {
+				margin-left: 0.9rem;
+				margin-right: 0;
+			}
+			:host([selectable]:not([disabled]):not([draggable]):hover) d2l-list-item-generic-layout,
+			:host([selectable]:not([disabled])) d2l-list-item-generic-layout.d2l-focusing,
+			:host([selectable][draggable]:not([disabled])) d2l-list-item-generic-layout.d2l-hovering {
+				background-color: var(--d2l-color-regolith);
+			}
+			:host([selected]:not([disabled])) d2l-list-item-generic-layout {
+				background-color: #F3FBFF;
```

Comment:
Normally we apply 0.5 opacity for disabled.  Does that not work in this case?  This appears to be a color that's not in our palette.

Reply:
Will dig into this more.

Reply:
This is actually when it isn't disabled but selected.